src/test/java/uk/gov/hmcts/reform/sendletter/logging/AppInsightsTest.java

```diff
@@ -49,10 +59,39 @@ public void should_fail_creating_app_insights_when_instrumentation_key_is_not_pr
 
     @Test
     public void should_create_app_insights_with_default_dev_mode_off() {
-        context.setInstrumentationKey("some-key");
+        context.setInstrumentationKey(IKEY);
 
         new AppInsights(telemetry);
 
         assertThat(TelemetryConfiguration.getActive().getChannel().isDeveloperMode()).isFalse();
     }
+
+    @Test
+    public void should_track_message_acknowledgement_event() {
+        context.setInstrumentationKey(IKEY);
+
+        AppInsights insights = new AppInsights(telemetry);
+        Map<String, String> properties = Collections.singletonMap("messageId", MESSAGE_ID);
+
+        insights.trackMessageAcknowledgement(MESSAGE_ID);
+
+        verify(telemetry).trackEvent(AppEvent.MESSAGE_ACKNOWLEDGED, properties, null);
+    }
+
+    @Test
+    public void should_track_message_received_event() {
+        context.setInstrumentationKey(IKEY);
+
+        Map<String, String> properties = new HashMap<>();
+
+        properties.put("service", SERVICE_NAME);
+        properties.put("template", TEMPLATE);
+        properties.put("messageId", MESSAGE_ID);
```

Comment:
you can use Guava's immutable map for nicer initialization

Reply:
`properties` doesn't say much, and the fact that this map is created in setup stage of the test doesn't help.
Perhaps it would be clearer if this was moved to verification part of the test? Even inlined if immutable map used